src/components/svg/Svg.css

```diff
@@ -0,0 +1,9 @@
+.svgWrapper {
```

Comment:
Does it make sense to have display inline-block and vertical-align middle here? What do you think?

Reply:
👍